tests/test_ansible.py

```diff
@@ -23,7 +23,7 @@ def check_docker_status(self):
    
     def check_exporter_status(self):
         """Helper to check exporter status"""
-        cmds=['ps -f -C  python2','ps -f -C  python3']
+        cmds=['ps -f -C  python3','ps -f -C  python3']
```

Comment:
Since both nvidia and net exporter use Python3 now should there still be a list of cmds?

Reply:
let me address

Reply:
Alright this has been addressed